iris/src/iris/web/routers/pipelines.py

```
@@ -319,7 +329,9 @@ def run_inconsistency_check_pipeline_worker(
             base_url=dto.execution.settings.artemis_base_url,
             initial_stages=dto.execution.initial_stages,
         )
-        pipeline = InconsistencyCheckPipeline(callback=callback)
+        pipeline = InconsistencyCheckPipeline(
+            callback=callback, local=dto.execution.settings.is_local()
+        )
     except Exception as e:
         logger.error("Error preparing inconsistency check pipeline: %s", e)
```

Ruff (0.14.8)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)